web_src/js/features/repo-diff.js

```diff
@@ -75,7 +77,8 @@ function initRepoDiffConversationForm() {
       }
       $newConversationHolder.find('.dropdown').dropdown();
       initCompReactionSelector($newConversationHolder);
-    } catch { // here the caught error might be a jQuery AJAX error (thrown by await $.post), which is not good to use for error message handling
+    } catch (error) {
+      console.error('Error:', error);
       showErrorToast(i18n.network_error);
```

Comment:
Note for later: This and similar errors should exclude `AbortError` from showing a toast. Extract the error handling into a function.

Reply:
Alternatively: Handle the `console.log` already in `GET`/`POST`, but re-throw the error in that case

Reply:
Possible, but re-throw might alter the error's stack trace. Need to check.

Network error can be detected via https://github.com/sindresorhus/is-network-error.

Reply:
Thought I do guess these should not be handled in the `request` function because that would likely lead the promise never resolving when the error is caught internally. It's best to handle such errors at the call site.